Rdmp.Core/Curation/Data/ExtractionInformation.cs

```diff
@@ -277,7 +277,7 @@ public override string GetSummary(bool includeName, bool includeID)
             sb.AppendLine($"Transforms Data: {FormatForSummary(IsProperTransform())}");
             sb.AppendLine(base.GetSummary(includeName, includeID));
 
-            return sb.ToString();
+            return sb.ToString().Replace("Is Primary Key:","Is Extraction Primary Key:");
```

Comment:
Noted that changing the property name of "IsPrimaryKey" would result in a database/api change so going with this for now.

Reply:
Made this a bit better in 
e1008d4